Dockerfile.ubi

```diff
@@ -201,15 +201,19 @@ WORKDIR /home/vllm
 
 ENTRYPOINT ["python3", "-m", "vllm.entrypoints.openai.api_server"]
 
-
 FROM vllm-openai as vllm-grpc-adapter
 
 USER root
 
 RUN --mount=type=cache,target=/root/.cache/pip \
     --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,from=build,src=/workspace/dist,target=/workspace/dist \
-    HOME=/root uv pip install $(echo dist/*.whl)'[tensorizer]' vllm-tgis-adapter==0.5.3
+    uv pip install $(echo /workspace/dist/*.whl)'[tensorizer]'  --verbose && \
```

Comment:
There's an issue in this Dockerfile at line 188 (not part of this PR) which prevents the dockerfile from building. You can either rebase on main or cherry pick 039e20956 to fix the issue.